service-base/pom.xml

```diff
@@ -109,13 +101,15 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-smallrye-health</artifactId>
+      <artifactId>quarkus-opentelemetry</artifactId>
```

Comment:
do we really need quarkus-telemetry here or just the opentelemetry artifacts?

Reply:
`quarkus-opentelemetry` is needed for the `io.quarkus.opentelemetry.runtime.tracing.TracerRuntimeConfig` class used in the  service-base `SamplerProducer` class.
The `SamplerProducer` class *could* be changed to use a plain `SmallRyeConfig` instance instead, but I would prefer the proper `TracerRuntimeConfig` object there.

Reply:
I see